src/utils.js

```diff
@@ -64,3 +64,25 @@ export function isEmpty(value) {
   }
   return false;
 }
+
+// We can have initial values undefined, but input makes them empty strings after edit/clear. We will consider those equal.
+
+const withException = (original, current) =>
+  (original === undefined || original === null) && current === ''
+    ? true
+    : undefined;
+
+function baseIsEqual(value, other) {
```

Comment:
At first i added some logic to check if value is array and if its empty, etc. But this can be provided with pristinify function.